lighthouse-core/report/v2/report-renderer.js

```diff
@@ -181,6 +182,39 @@ class ReportRenderer {
   }
 
   /**
+   * @param {string} text
+   * @return {!HTMLSpanElement}
+   */
+  _convertMarkdownLinksToElement(text) {
+    const element = this._createElement('span');
+
+    // Split on markdown links (e.g. [some link](https://...)).
+    const parts = text.split(/\[(.*?)\]\((https?:\/\/.*?)\)/g);
+
+    while (parts.length) {
+      // Pop off the same number of elements as there are capture groups.
+      const [preambleText, linkText, linkHref] = parts.splice(0, 3);
+      element.appendChild(this._document.createTextNode(preambleText));
+
+      // Append link if there are any.
+      if (linkText && linkHref) {
+        const a = this._createElement('a');
+        a.rel = 'noopener';
+        a.target = '_blank';
+        a.textContent = linkText;
+        try {
+          a.href = (new URL(linkHref)).href;
+          element.appendChild(a);
+        } catch (e) {
+          element.appendChild(this._document.createTextNode(linkText));
```

Comment:
but why is the link busted? Right now, at least, they're all coming from helpText. An error in one of those should be fixed immediately